apps/fluent-tester/src/TestComponents/Badge/PresenceBadgeTest.android.tsx

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+import React, { useState } from 'react';
+import { View, Platform, Text } from 'react-native';
+
+import { ToggleButton } from '@fluentui/react-native';
+import { PresenceBadge } from '@fluentui-react-native/badge';
+
+export const PresenceBadgeTest: React.FunctionComponent = () => {
+  const [outOfOffice, setOutOfOffice] = useState(false);
+  const svgIconsEnabled = ['ios', 'macos', 'win32', 'android'].includes(Platform.OS as string);
```

Comment:
This is android specific file i don't this is needed anymore here. We can remove the conditionals.